Review/Review.cs

```diff
@@ -246,14 +246,14 @@ public class Review {
         _BaseReport.scienceLab = VesselHelper.VesselHasModuleAlias (Base, "ScienceLab");
         _BaseReport.fuel = VesselHelper.GetResourceCount (Base, "LiquidFuel");
         _BaseReport.ore = VesselHelper.GetResourceCount (Base, "Ore");
-        _BaseReport.entity = Base.landedAt;
+        _BaseReport.entity = Base.mainBody.name;
```

Comment:
Is `MainBody` going to be the body the base is at or is it going to be Kerbin?
It's been a while since I've been in here.